tests/modeling/test_rpn.py

```diff
@@ -77,6 +79,68 @@ def test_rpn(self):
             )
             self.assertTrue(torch.allclose(proposal.objectness_logits, expected_objectness_logit))
 
+    @unittest.skipIf(TORCH_VERSION < (1, 6), "Insufficient pytorch version")
+    def test_rpn_scriptability(self):
+        torch.manual_seed(121)
```

Comment:
The expected output was tested in the method above already. In this test we can just use random inputs and only test that the two RPN computes identical results (i.e. no need to compare with `expected_*` again).

Reply:
done, now i just compare the outputs of the two RPN.

Reply:
no longer need to fixed the seed, since we don't have a fixed expected output any more